src/libs/disk-utils.js

```diff
@@ -1,21 +0,0 @@
-import { Fragment } from 'react'
```

Comment:
Deleted this file because it is unused and we ended up not using the deletePDText in our delete work, so seemed not useful at this point.

Reply:
👍